File: output/ChatGPT/codenet/Python/C++/s018260659.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    
    int cnt = 1;
    while (cnt <= 9) {
        if (N % cnt == 0 && 1 <= N / cnt <= 9) {
            cout << "Yes" << endl;
            break;
        } else {
            cnt++;
        }
    }
    
    if (cnt > 9) {
        cout << "No" << endl;
    }
    
    return 0;
}

File: output/ChatGPT/codenet/Python/Java/s018260659.java
int N = Integer.parseInt(input);
int cnt = 1;
while (cnt <= 9) {
    if (N%cnt == 0 && 1 <= N/cnt && N/cnt <= 9) {
        System.out.println("Yes");
        break;
    }
    else {
        cnt += 1;
    }
}
if (cnt > 9) {
    System.out.println("No");
}